src/main/backend/setup.ts

```diff
@@ -315,13 +313,23 @@ export const setupBackend = async (
     token: ProgressToken,
     useSystemPython: boolean,
     systemPythonLocation: string | undefined | null,
-    hasNvidia: () => Promise<boolean>
+    hasNvidia: () => Promise<boolean>,
+    getRootDir: () => Promise<string>
 ): Promise<BackendProcess> => {
     token.submitProgress({ totalProgress: 0 });
 
+    const currentExecutableDir = path.dirname(app.getPath('exe'));
+    log.info(`currentExecutableDir setupBackend: ${currentExecutableDir}`);
```

Comment:
You might want to use the variable in `platform.ts`.

Reply:
oops i forgot to remove that log

Reply:
both those lines were from debugging